File: reimi/frontend/reimi_app/lib/domain/value_objects/feeling_type.dart
import 'package:freezed_annotation/freezed_annotation.dart';

enum FeelingType {
  @JsonValue('VERY_COLD')
  veryCold, // 極寒
  @JsonValue('COLD')
  cold, // 寒い
  @JsonValue('COMFORTABLE')
  comfortable, // ちょうどいい
  @JsonValue('WARM')
  warm, // 暖かい
  @JsonValue('NOT_APPLICABLE')
  notApplicable, // あてはまらない
}

File: reimi/backend/reimi_app/src/main/java/com/reimi/reimi_app/domain/model/report/value/FeelingType.java
package com.reimi.reimi_app.domain.model.report.value;

public enum FeelingType {

    VERY_COLD("極寒"),
    COLD("寒い"),
    COMFORTABLE("ちょうどいい"),
    WARM("暖かい"),
    NOT_APPLICABLE("あてはまらない");

    private final String label;

    FeelingType(String label) {
        this.label = label;
    }

    public String getLabel() {
        return label;
    }

}